sktime/transformations/multiplexer.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements forecaster for selecting among different model classes."""
+
+from sklearn.base import clone
+
+from sktime.base import _HeterogenousMetaEstimator
+from sktime.transformations._delegate import _DelegatedTransformer
+from sktime.transformations.base import BaseTransformer
+
+__author__ = ["miraep8"]
```

Comment:
Also, don't forget to update `CODEOWNERS` and `docs/source/api_reference/transformations.rst` :)

Reply:
Thank you for the reminder on the CODEOWNERS bit.  And I actually haven't changed the docs directly like this before, so the pointer was really helpful.  I will leave this as unresolved until I can check out how the read the docs build looks after the changes. :)